content/en/blog/ospo-definition.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Open Source Program Office (OSPO) Definition"
+author: todogroup
+date: 2020-05-13
+---
+
+The TODO Group are [over 5 years old](https://todogroup.org/blog/welcome/) and as an organization have brought together the brightest in the field of open source program management. We have produced a variety of open source program management [guides](https://todogroup.org/guides/), [tools](https://github.com/todogroup/repolinter), [surveys](https://github.com/todogroup/survey) and even an [awesome list](https://github.com/todogroup/awesome-oss-mgmt) to advance the practice.
+
+Today, we are formalizing the [definition](https://github.com/todogroup/ospodefinition.org) of what an open source program (colloquially an OSPO) is based on our years of experience and to ensure we have a common lexicon in the industry when we describe open source programs:
+
+*An open source program office (OSPO) is designed to be the center of competency for an organization's open source operations and structure. This can include training developers, ensuring legal compliance, engaging with and building communities, and defining policies that govern code usage, distribution, selection, auditing and more.*
```

Comment:
s/center of competency/center of gravity

Competency comes with experience and time, and in the beginning, it'll be more learning than knowing. A center of gravity draws people together, and sounds more inclusive of folks who are earlier in their OSPO journey, who don't know the way just yet.

Reply:
thanks!